tests/e2e/machine_pool_test.go

```diff
@@ -47,22 +45,24 @@ var _ = Describe("TF Test", func() {
 		})
 		Context("Author:amalykhi-High-OCP-64757 @OCP-64757 @amalykhi", func() {
 			It("Create a second machine pool", func() {
-				token := os.Getenv(con.TokenENVName)
+				By("Create a second machine pool")
 				replicas := 3
 				machine_type := "r5.xlarge"
-				name := "testmp14"
+				name := "ocp-64757"
 				MachinePoolArgs := &exe.MachinePoolArgs{
 					Token:       token,
-					Cluster:     cluster_id,
+					Cluster:     clusterID,
 					Replicas:    replicas,
 					MachineType: machine_type,
 					Name:        name,
 				}
 
 				err := mpService.Create(MachinePoolArgs)
+				Expect(err).ToNot(HaveOccurred())
+				By("Verify the parameters of the created machinepool")
 				mp_out, err := mpService.Output()
 				Expect(err).ToNot(HaveOccurred())
-				mp_response, err := cms.RetrieveClusterMachinePool(conn.RHCSConnection, cluster_id, name)
+				mp_response, err := cms.RetrieveClusterMachinePool(conn.RHCSConnection, clusterID, name)
```

Comment:
Please create a new function and place those lines in it, as they are repeated in all the test cases.
```
				mp_response, err := cms.RetrieveClusterMachinePool(conn.RHCSConnection, clusterID, name)
				Expect(err).ToNot(HaveOccurred())
				Expect(mp_response.Status()).To(Equal(http.StatusOK))
				return mp_response.Body()
```

Reply:
@bardielle Please, review the changes. Is that what you referred to?